dali/operators/reader/numpy_reader_gpu_op.cc

```diff
@@ -97,10 +97,10 @@ void NumpyReaderGPU::Prefetch() {
 
   const size_t kGDSChunkGranularity = 1<<20;
 
-  size_t chunk_size = align_up(div_ceil(static_cast<uint64_t>(curr_tensor_list.total_nbytes()),
+  size_t chunk_size = align_up(div_ceil(static_cast<uint64_t>(curr_tensor_list.chunks_nbytes()),
```

Comment:
chunks_nbytes returns a vector - how is that supposed to work?

Reply:
This is from an in-between commit.